test/functional/helper/AccessTokenRequester.groovy

```diff
@@ -16,7 +16,14 @@ class AccessTokenRequester {
     }
 
     static String getAccessToken(Map params) {
-        def response = requestAccessToken(params)
+        params = params.clone() // Don't mess with the original Map
+        String clientId = params.remove('client_id')
+        String clientSecret = params.remove('client_secret') ?: ''
+
+        def response = clientId != null ?
```

Comment:
Did you also update `requestAccessToken`? I saw 13 failed tests caused by the `Set-Cookie` header when the `NullRememberMeServices` was not used.